contracts/script/multisigTransactionProposals/README.md

```diff
@@ -0,0 +1,91 @@
+# Proposing Multisig Transactions via the Safe SDK
+
+The [Safe SDK](https://github.com/safe-global/safe-core-sdk/blob/main/guides/integrating-the-safe-core-sdk.md) is being
+used to propose transactions that only the Safe multisig admin wallet can perform. The proposer of these transactions is
+also part of the multisig wallet but is used to orchestrate the process. E.g. If you require 3 of 5 trusted signers to
+sign a transaction, then the multisig wallet should require 4 of 5 signers where there 4th signer is the orchestrator
+wallet.
+
+## Set Permissioned Prover
+
+To enable the permissioned prover on the light client contract, ensure that the following environment variables are set
+in the `.env` file:
+
+- `SEPOLIA_RPC_URL`
+- `SAFE_ORCHESTRATOR_PRIVATE_KEY`
```

Comment:
@alysiahuggins where can I get the value for this?

Reply:
Is it the private key of the orchestrator ethereum wallet?

Reply:
If yes, how does this work with hardware wallets if one were to use that?

Reply:
I guess we have to change some code, but I think what we currently have is fine. We could consider using mnemonic and account index index of private key because that is what we use everywhere else for ethereum wallets.